extensions/keka/src/utils/applescript-utils.ts

```diff
@@ -0,0 +1,35 @@
+import { showFailureToast, runAppleScript } from "@raycast/utils";
+
+export const scriptCompressFiles = async (filePaths: string[]) => {
+  scriptExecuteFiles("compress", filePaths);
+};
+
+export const scriptExtractFiles = async (filePaths: string[]) => {
+  scriptExecuteFiles("extract", filePaths);
+};
+
+export const scriptSendFiles = async (filePaths: string[]) => {
+  scriptExecuteFiles("send", filePaths);
+};
+
+export const scriptExecuteFiles = async (
+  action: string,
+  filePaths: string[],
+) => {
+  try {
+    const appleScriptFileList = filePaths
+      .map((path) => `POSIX file "${path}"`)
+      .join(", ");
+
+    const script = `
+		tell application id "com.aone.keka"
+		${action} {${appleScriptFileList}}
+		end tell
+		`;
+
+    await runAppleScript(script);
+  } catch (e) {
+    console.error(String(e));
+    await showFailureToast(e, { title: "Failed to execute" });
```

Comment:
```suggestion
    await showFailureToast(e, { title: e instanceof Error ? e.toString() : "Failed to execute" });
```

Reply:
Let's expose the actual error message to users. And ditto for other commands.

Reply:
Done!

I thought the error was always shown like this, which is by the way pretty cool:

<img width="752" alt="Screenshot 2025-05-21 at 21 52 31" src="https://github.com/user-attachments/assets/c15ef464-fcfd-42a5-ad70-361348b0d39d" />

I see now that if the main window is closed (error after `closeMainWindow`) it's shown as a dialog and only the title is shown. Maybe that dialog can be enhanced in the future to show the error below in a smaller, lighter font.

![image](https://github.com/user-attachments/assets/f0930c9e-1d8b-4eca-97ae-dbc90e5b64a1)